lib/components/visualization/tables/SortButton.js

```diff
@@ -63,8 +63,10 @@ var SortButton = React.createClass({
 	 * @return {jsx}
 	 */
 	render: function () {
+		let sort_icon = (this.state.direction === 'asc') ? 'icon-chevron-up' : 'icon-chevron-down';
```

Comment:
Check out it out [here](https://github.com/JedWatson/classnames)

It's a really useful way to conditionally set classes :) 

Let me know if you have any questions.